mqtt/README.md

```diff
@@ -10,30 +10,31 @@ The service is configured using the environment variables presented in the
-| MF_MQTT_ADAPTER_WS_TARGET_HOST    | MQTT broker host for MQTT over WS                      | localhost             |
-| MF_MQTT_ADAPTER_WS_TARGET_PORT    | MQTT broker port for MQTT over WS                     | 8080                  |
-| MF_MQTT_ADAPTER_WS_TARGET_PATH    | MQTT broker MQTT over WS path                          | /mqtt                 |
-| MF_MQTT_ADAPTER_FORWARDER_TIMEOUT | MQTT forwarder for multiprotocol communication timeout | 30s                   |
-| MF_NATS_URL                       | NATS broker URL                                        | nats://127.0.0.1:4222 |
-| MF_THINGS_AUTH_GRPC_URL           | Things gRPC endpoint URL                               | localhost:8181        |
-| MF_THINGS_AUTH_GRPC_TIMEOUT       | Timeout in seconds for Things service gRPC calls       | 1s                    |
-| MF_JAEGER_URL                     | URL of Jaeger tracing service                          | ""                    |
-| MF_MQTT_ADAPTER_CLIENT_TLS        | gRPC client TLS                                        | false                 |
-| MF_MQTT_ADAPTER_CA_CERTS          | CA certs for gRPC client TLS                           | ""                    |
-| MF_MQTT_ADAPTER_INSTANCE          | Instance name for event sourcing                       | ""                    |
-| MF_MQTT_ADAPTER_ES_URL            | Event sourcing URL                                     | localhost:6379        |
-| MF_MQTT_ADAPTER_ES_PASS           | Event sourcing password                                | ""                    |
-| MF_MQTT_ADAPTER_ES_DB             | Event sourcing database                                | "0"                   |
-| MF_AUTH_CACHE_URL                 | Auth cache URL                                         | localhost:6379        |
-| MF_AUTH_CACHE_PASS                | Auth cache password                                    | ""                    |
-| MF_AUTH_CACHE_DB                  | Auth cache database                                    | "0"                   |
+| Variable                                 | Description                                            | Default               |
+|------------------------------------------|--------------------------------------------------------|-----------------------|
+| MF_MQTT_ADAPTER_LOG_LEVEL                | mProxy Log level                                       | error                 |
+| MF_MQTT_ADAPTER_MQTT_PORT                | mProxy port                                            | 1883                  |
+| MF_MQTT_ADAPTER_MQTT_TARGET_HOST         | MQTT broker host                                       | 0.0.0.0               |
+| MF_MQTT_ADAPTER_MQTT_TARGET_PORT         | MQTT broker port                                       | 1883                  |
+| MF_MQTT_ADAPTER_MQTT_TARGET_HEALTH_CHECK | URL of broker health check                             | ""                    |
```

Comment:
Is this default value - empty string? I thought it was `http://0.0.0.0:8888/health`. BTW, maybe you should consider `localhost` for consistency?

Reply:
Default is empty.
In adapter there is `MF_MQTT_ADAPTER_MQTT_TARGET_HEALTH_CHECK` that is empty by default, because when you start mainflux natively (`make rundev`) you don't have broker started, so you don't know if it's going to be verne or mosquitto or some other broker.
In docker-compose `MF_MQTT_ADAPTER_MQTT_TARGET_HEALTH_CHECK` gets value of `MF_MQTT_BROKER_HEALTH_CHECK` . That other envvar has default value `http://0.0.0.0:8888/health`
So, only when you start docker composition (`make run`) you run mainflux with VerneMQ  and also check its health

Reply:
Since we have all the `MQTT_TARGET` env vars default values set to our default MQTT broker, I think that health check should not be different.

Reply:
That example is actually very similiar with my solution.
Target host by default is hardcoded to `0.0.0.0` in [main.go](https://github.com/mainflux/mainflux/blob/master/cmd/mqtt/main.go#L40) and then it's overwritten in [docker-compose](https://github.com/mainflux/mainflux/blob/master/docker/docker-compose.yml#L275) to `vernemq`
I just didn't hardocode values, I think it's better to use env vars, but same principle. In main it's empty string, in docker-compose (where you use `vernemq` as target host) health-check get value of VerneMQ health check endpoint.

Reply:
On the other hand, it's not a good example because whatever broker you choose it will have default mqtt endpoint on localhost:1883. But health-check endpoint on `:8888/health` is something that is specific just for VerneMQ, this is why I think it's better to be initialized only in places where we use VerneMQ, like docker-compose or helm charts

Reply:
Actualy, this was error, in docker composition address of VerneMQ container is not `0.0.0.0` but `vernemq`. Fixed.